test/transaction-runner.ts

```diff
@@ -306,6 +334,87 @@ describe('TransactionRunner', () => {
             done();
           });
       });
+
+      describe('when multiplexed session is enabled for read/write', () => {
+        it('should update the multiplexedSessionPreviousTransactionId before retrying aborted transaction', async () => {
```

Comment:
There are a lot of test cases in `spanner.ts` like `should retry UNAVAILABLE during streaming with txn ID from inline begin response` where you can check the retry behavior and test if the `multiplexedSessionPreviousTransactionId` is set correctly

Reply:
Yeah there are, but those are end to end tests and we haven't added support in the public methods yet

I will keep a note of this, and will add the test in the upcoming PRs